app/src/main/java/org/mozilla/focus/tabs/Tab.java

```diff
@@ -180,6 +180,10 @@ public void detach() {
     }
 
     /* package */ TabView createView(@NonNull final Activity activity) {
+        return createViewWithFallbackUrl(activity, null);
+    }
+
+    /* package */ TabView createViewWithFallbackUrl(@NonNull final Activity activity, final String url) {
```

Comment:
I would suggest that rename `createView` to `initializeView`. We don't need pass url as parameter since the url do come from this tab itself.

```java
 /* package */ TabView initializeView(@NonNull final Activity activity) {
        final String url = this.getUrl();
```

Reply:
Need to discuss with you.